### DISCUSSION TITLE
action link to whats app number

### DISCUSSION THREAD

**Discussion by ridone713** [filamentphp/filament#11951] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

How do you create a table action that leads directly to the WhatsApp number in the table so you can contact it directly in the WhatsApp application accompanied by a message template?

**Comment by ArtMin96** [comment]
```php
Action::make()
    ->url(fn (User $record): string => "https://wa.me/{$user->phone}")
    ->openUrlInNewTab(),
```
Something like this. But this is a question that you can easily google

**Reply by ridone713** [reply]
Action::make('wa')->label('WA')->color('success') ->url(fn (Penghuni $record): string => "https://wa.me/{$penghuni->no_hp}")
                ->openUrlInNewTab(),

thanks for the answer and i've been modified your code but it's still not working

**Reply by leandrocfe** [reply]
```php
Tables\Actions\Action::make('wa')
    ->label('WA')
    ->color('success')
    ->url(fn (Penghuni $record): string => "https://wa.me/{$record->no_hp}")
    ->openUrlInNewTab()
```
